tapir/shifts/tests/test_shift_template_duplicate.py

```diff
@@ -48,3 +48,65 @@ def test_duplicate_shift_template(self):
             self.assertEqual(
                 template.slot_templates.name, shift_template.slot_templates.name
             )
+
+    def test_shiftTemplateDuplicateForm_duplicateSingleTemplate(self):
```

Comment:
We need to test that all the fields are copied correctly, including start_date, start_time...